### ISSUE TITLE
city and state are null

### ISSUE THREAD

**Issue by lakshminarayanan002** [john-doherty/offline-geo-from-ip#15]
There is an issue for most of ip city and state are set null in response

**Comment by john-doherty** [comment]
Yes, there is an issue mentioned [here](https://www.npmjs.com/package/maxmind-db-reader).
> Most IP's don't seem to have city data in the GeoLite2 City database. Only american cities are present, if your IP is american but not returning a city try to replace the last number of your IP by a 0 (only with a IPv4 address. for IPv6 address it doesn't seem to work).

I've pushed a [new version](https://github.com/john-doherty/offline-geo-from-ip/blob/master/index.js#L18) that attempts to improve on this. Reopen if it still does not work for you
